### PR TITLE
i18n(es): Updated `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content
- 
#### Description

- What does this PR change? Give us a brief description.
Updated `view-transitions.mdx` based on this commit 83627419e21f6927dbc1856d38e72c0955e7dfa0 and added missing part.

### CODE REVIEW

File: src/content/docs/es/guides/view-transitions.mdx [hunk 1]
```diff
@@ -200,6 +200,62 @@ import { ViewTransitions } from 'astro:transitions';
...
+Un evento que se dispara al final de la navegación de la página, después de que la nueva página sea visible para el usuario y se carguen los estilos y scripts de bloqueo. Puedes escuchar este evento en el `document`.
+
+El componente `<ViewTransitions />` dispara este evento tanto en la navegación inicial de la página (MPA) como en cualquier navegación posterior, ya sea hacia adelante o hacia atrás.
+
+Puedes usar este evento para ejecutar código en cada navegación de página, o solo una vez:
+
+```astro "{ once: true }"
+<script>
+  document.addEventListener('astro:load', () => {
+    // This only runs once.
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:230]:
```suggestion
    // Sólo se ejecuta una vez.
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 2]
```diff
@@ -200,6 +200,62 @@ import { ViewTransitions } from 'astro:transitions';
...
+
+```astro
+<script>
+	const setDarkMode = () => {
+		if (localStorage.darkMode) {
+			document.documentElement.dataset.dark = '';
+		}
+	};
+
+  // Runs on initial navigation
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:252]:
```suggestion
  // Se ejecuta en la navegación inicial
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 3]
```diff
@@ -200,6 +200,62 @@ import { ViewTransitions } from 'astro:transitions';
...
+	setDarkMode();
+  // Runs on view transitions navigation
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:254]:
```suggestion
  // Se ejecuta en la navegación de transiciones de vista
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**